jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/codeeditor/JmixCodeEditor.java

```diff
@@ -65,79 +68,140 @@ protected void initComponent() {
         }
 
         setInvalid(false);
+        setPrintMarginColumn(PRINT_MARGIN_COLUMN_DEFAULT_VALUE);
+        setFontSize(FONT_SIZE_DEFAULT_VALUE);
+
         addValueChangeListener(e -> validate());
         addClientValidatedEventListener(e -> validate());
     }
 
+    /** @see JmixCodeEditor#setHighlightActiveLine(boolean)
+     *
+     * @return Visibility of highlighting the active line
+     */
     @Synchronize(PROPERTY_HIGHLIGHT_ACTIVE_LINE)
     public boolean isHighlightActiveLine() {
         return getElement().getProperty(PROPERTY_HIGHLIGHT_ACTIVE_LINE, true);
     }
 
+    /**
+     * Sets the highlight of the line the cursor is on
+     */
     public void setHighlightActiveLine(boolean highlightActiveLine) {
         getElement().setProperty(PROPERTY_HIGHLIGHT_ACTIVE_LINE, highlightActiveLine);
     }
 
+    /**
+     * @see JmixCodeEditor#setShowGutter(boolean)
+     * @return Gutter visibility
+     */
     @Synchronize(PROPERTY_SHOW_GUTTER)
     public boolean isShowGutter() {
         return getElement().getProperty(PROPERTY_SHOW_GUTTER, true);
     }
 
+    /**
+     * Sets visibility of the Gutter
+     */
     public void setShowGutter(boolean showGutter) {
         getElement().setProperty(PROPERTY_SHOW_GUTTER, showGutter);
     }
 
+    /**
+     * @see JmixCodeEditor#setShowLineNumbers(boolean)
+     * @return Editor line numbering visibility
+     */
     @Synchronize(PROPERTY_SHOW_LINE_NUMBERS)
     public boolean isShowLineNumbers() {
         return getElement().getProperty(PROPERTY_SHOW_LINE_NUMBERS, true);
     }
 
+    /**
+     * Sets the line numbering of the editor
```

Comment:
I suppose that you set line numbers visibility here... you know because of boolean